VIDEO_6_SCRIPT.md

```diff
@@ -0,0 +1,54 @@
+
+## How AsyncAPI approaches this concept
+
+Transparency has probably become the fundamental concept in the democratization processes undertaken by governments, institutions, companies, organizations, projects... in recent years. In its ideal state, democracy is nothing more than a method that allows the community members to analyze reality from different points of view to make decisions finally.
+
+The open-source policy has come in the last decades, meeting all the characteristics and fulfilling the gaps for developing a real and practical democracy. Open-source appears as a system that makes information free and at the same time allows users to see their management and decision-making processes. In short, everyone is invited to collaborate. In the end, a community arises, and it is built collectively, with every user as an active member of it.
+
+AsyncAPI arises precisely from this basis. As an open government project, it advocates transparency and democratization in its system of governance and organization. This organizational system aims to empower people based on a horizontal decision-making structure.
+
+All relevant project data are public. The developed tasks are accessible and open to the participation of the community. This is intended to cultivate an environment of trust and respect for the user that encourages their involvement in pursuing a common goal. Collaboration and participation are key elements of a never-ending dynamic, an adaptable and flexible process where nothing is imposed or taken for granted.
+
+## Examples of tools that promote transparency in project work
+
+As mentioned above, the project is managed by all the people who actively collaborate in its community. AsyncAPI has several tools that ensure transparency in project management. Let's talk about them:
+
+- **Open governance**: To ensure that the project is managed by its community and that it’s the one pulling the strings, last year, it was hosted by an open government foundation, Linux Foundation. Hosting the project on a neutral foundation ensures true democratization for the project and its community. At the same time, it provides the long-term sustainability and success of the initiative. 
+
+- **Economy**: all the financial management of the project is managed through Open Collective. It’s an online crowdfunding platform for communities that provides the necessary tools to raise and share the finances raised with complete transparency. You can find out where the money comes from in this space and its use.
+
+- **Transparent communication and work**: To ensure that the principle of transparency governs communication, everything is said and done in public. The community uses Slack as the first communication platform, and secondly, GitHub is used as a means of communication and work. Therefore, all information about any project management is public, no matter how insignificant or essential it may be.
```

Comment:
Github is first communication channel and this is why we make all decisions there, Slack is only for sync communication enablement and keeping the closer bond with the community on daily basis.